Developing-with-Minikube.adoc

```diff
@@ -0,0 +1,157 @@
+= Developing riff with Minikube
+
+== Prerequisites
+
+* A running 1.7+ Kubernetes cluster
+
+TIP: If using Minikube, we recommend using Minikube v0.23.0 or v0.24.1 and avoiding v0.24.0 due to some DNS issues
+
+* Kubernetes CLI `kubectl` installed and on the local system PATH. We recommend using the same version or later as the Kubernetes cluster you are using.
+
+* Docker, you will need Docker installed. We have used Docker version 17.x or later.
+
+* Helm, you need helm installed, see instructions link:Getting-Started.adoc#helm[here]. 
```

Comment:
This guide is very Helm focused, which IMO doesn't reflect the title "__developing__ with minikube". I think the contents of "manual build and deploy" is more appropriate for this.
This guide is more about "how to tweak a single component of the helm chart here and there", which as a matter of fact may be needed in a production setup.

WDYT?

Reply:
Yes, would agree, changing titles and some content to reflect this